templates/common/_base/units/ovsdb-server.service.yaml

```diff
@@ -5,3 +5,6 @@ dropins:
     contents: |
       [Service]
       Restart=always
+      ExecStartPre=-/bin/sh -c '/usr/bin/chown -R ${OVS_USER_ID} /var/lib/openvswitch'
+      ExecStartPre=-/bin/sh -c '/usr/bin/chown -R ${OVS_USER_ID} /etc/openvswitch'
+      ExecStartPre=-/bin/sh -c '/usr/bin/chown -R ${OVS_USER_ID} /run/openvswitch'
```

Comment:
We have to check if upgrades from 4.9 to 4.10 are still affected by this. If not, this commit can be dropped.